### DISCUSSION TITLE
need tutorial

### DISCUSSION THREAD

**Discussion by ahmedRSA** [charmbracelet/bubbletea#614] [General]
need a tutorial explaining everything
complete beginner in TUI stuff 
video tutorial will be great if anyone have pls share

**Comment by jonfriesen** [comment]
Trying starting here: [Golang TUI Project Basics - Shopping List | Part I](https://youtu.be/5lxQJS3b38w). Then checkout the [@charmcli](https://www.youtube.com/@charmcli) youtube channel.

**Comment by bashbunni** [comment]
I'll make note of this for our upcoming youtube videos :)

**Comment by bashbunni** [comment]
Hey @ahmedRSA now that we've got a few tutorials on our [YouTube](https://charm.sh/yt) channel, have you become more comfortable with the concepts, or is there something still confusing you? I can aim to clarify other aspects in future tutorials on our channel. Let us know!

**Reply by ahmedRSA** [reply]
i created some projects with it but its hard to pick up as a beginner but once you understand core concepts its easy.

**Reply by bashbunni** [reply]
@ahmedRSA I felt the same way. I feel like it just takes some time to get used to... Let me know if you have any feedback for ways we can make it easier to onboard. :smile:
